requirements.txt

```diff
@@ -1,24 +1,31 @@
-asgiref==3.8.1
-astroid==3.3.4
-click==8.1.7
-colorama==0.4.6
-cssbeautifier==1.15.1
-dill==0.3.9
-Django==5.1.1
-djlint==1.35.2
-EditorConfig==0.12.4
-html-tag-names==0.1.2
-html-void-elements==0.1.0
-isort==5.13.2
-jsbeautifier==1.15.1
-json5==0.9.25
-mccabe==0.7.0
-pathspec==0.12.1
-platformdirs==4.3.6
-pylint==3.3.1
-PyYAML==6.0.2
-regex==2024.9.11
-six==1.16.0
-sqlparse==0.5.1
-tomlkit==0.13.2
-tqdm==4.66.5
+Package            Version
+------------------ ---------
+asgiref            3.8.1
+astroid            3.3.4
```

Comment:
Where did this format for packages come from ? my `pip install` doesn't work with this format (and neither does the automated pipeline)

Reply:
(as opposed to the `package==version` style)

Reply:
ah, did a git list > requirements.txt instead of git freeze > requirements.txt my bad